### ISSUE TITLE
Oculus Shaders compatibility?

### ISSUE THREAD

**Issue by Its-Just-Luci** [Sinytra/Connector#278]
### Describe the bug

Oculus is crashing when installed alongside my other mods

### Steps to reproduce

Boot game and it crashes on load

### Logs

<script src="https://gist.github.com/Its-Just-Luci/4affd423cb77f2e3a62728ad4aa10c23.js"></script>

### Additional context

It seems to be a Sodium-Sinytra-Oculus compatibility issue :/

**Comment by lSherozl** [comment]
> ### Describe the bug
> Oculus is crashing when installed alongside my other mods
> 
> ### Steps to reproduce
> Boot game and it crashes on load
> 
> ### Logs
> <script src="https://gist.github.com/Its-Just-Luci/4affd423cb77f2e3a62728ad4aa10c23.js"></script>
> ### Additional context
> It seems to be a ~Sodium~ Rubidium-Sinytra-Oculus compatibility issue :/

I have rubidium and it works fine in the prominence modpack, you should list your mods, from what I know connector for me isn't conflicting rubidium

**Comment by Its-Just-Luci** [comment]
yeah rubidium works fine, it's when adding oculus that the game crashes, sorry should have been more specific

**Comment by Su5eD** [comment]
> oculus-mc1.19.2-1.6.4.jar

This says you're using oculus for mc 1.19.2
